cmd/serverless-init/main_test.go

```diff
@@ -36,3 +39,28 @@ func TestProxyLoaded(t *testing.T) {
 	assert.Equal(t, proxyHttp, proxyHttpConfig)
 	assert.Equal(t, proxyHttps, proxyHttpsConfig)
 }
+
+func TestTagsSetup(t *testing.T) {
+	t.Setenv("DD_TAGS", "key1:value1 key2:value2 key3:value3")
+	t.Setenv("DD_EXTRA_TAGS", "key22:value22 key23:value23")
+	tags := map[string]string{
+		"key1":  "value1",
+		"key2":  "value2",
+		"key3":  "value3",
+		"key22": "value22",
+		"key23": "value23",
+	}
+	_, _, _, metricAgent := setup()
+	assert.True(t, tagArrayContainsTags(metricAgent.GetExtraTags(), tags))
```

Comment:
Would `assert.ElementsMatch` work?

Reply:
Unfortunately not. There are extra tags that are being added, such as `datadog_init_version`, so the need is "contains", not "exact match"

Reply:
Then maybe `assert.Subset`?

Reply:
Then maybe `assert.Subset`?

Reply:
Oh great! That's perfect, thanks!